core-httpclient-impl/src/main/java/com/optimizely/ab/odp/DefaultODPApiManager.java

```diff
@@ -206,17 +205,19 @@ public void sendEvents(String apiKey, String apiEndpoint, List<ODPEvent> events)
             response = httpClient.execute(request);
         } catch (IOException e) {
             logger.error("Error retrieving response from event request", e);
-            return;
+            return null;
```

Comment:
Should we retry on "null" response?  How can we differentiate connection error (retry) from other permanent errors?